docs/README.rst

```diff
@@ -1,5 +1,13 @@
+Install extra python requirements to be able to build html docs locally::
+
+    pip install sphinx sphinx_rtd_theme -r requirements.txt
```

Comment:
I wonder if we should add sphinx and sphinx_rtd_theme to docs/requirements.txt

Reply:
I wondered the same - I thought that ``requirements.txt`` are used by RTD to install extra requirements needed to build the docs. Although just adding sphinx and sphinx_rtd_theme should do no harm if they are already installed - let me do this and if something breaks on RTD we can revert/fix it.